crates/ironrdp-web/src/session.rs

```diff
@@ -603,6 +603,7 @@ impl Session {
                             hotspot_y,
                         })?;
                     }
+                    ActiveStageOutput::DeactivateAll(_) => todo!("DeactivateAll"),
```

Comment:
issue: Can’t we handle this in a more graceful manner?

Reply:
Presuming [this system](https://github.com/Devolutions/IronRDP/pull/421/files#diff-33dc7055fcffcb2318664572070611527a75668ad24c01e2ce01ae83e0b27defR609-R655) is confirmed to work in web (just waiting for another test from Nefix in matrix), I think I can safely copy that into here.